### ISSUE TITLE
Type Annotation Wrong on StakeInfo.list_of_tuple_from_vec_u8

### ISSUE THREAD

**Issue by thewhaleking** [camfairchild/bittensor#1845]
### Describe the bug

There is a type annotation error in `StakeInfo.list_of_tuple_from_vec_u8`:

```python
    @classmethod
    def list_of_tuple_from_vec_u8(
        cls, vec_u8: List[int]
    ) -> Dict[str, List["StakeInfo"]]:
        r"""Returns a list of StakeInfo objects from a ``vec_u8``."""
        decoded: Optional[
            List[Tuple(str, List[object])]
        ] = from_scale_encoding_using_type_string(
            input=vec_u8, type_string="Vec<(AccountId, Vec<StakeInfo>)>"
        )
        ...
```
Specifically: `List[Tuple(str, List[object])]`.

This should be fixed, and would be a good first issue.

### To Reproduce

Look at line 832 of `chain_data.py`.

### Expected behavior

`List[Tuple[str, List[object]]]`

### Environment

Any

**Comment by opendansor** [comment]
commit 29efad2f has already addressed this change.

https://github.com/opentensor/bittensor/pull/1840/files#diff-dc51391a180335c2350c6f5bcb125fc905cb18ee2abece7ff01ce32ccc1f58acL832

We can go ahead and close the issue.

**Comment by thewhaleking** [comment]
Fixed by #1840
